### PR TITLE
WIP: Add server nickname

### LINKED ISSUE #288: [Feature] Add support for Server Nickname
Having the entire servername in OOC is a bit cumbersome and Tsu did it better there.

### PR DESCRIPTION
**Pull request by cow-face**
Resolves #288 

Not sure what to put as the default nickname in the sample config.

### CODE REVIEW

**Comment by scatterflower** [comment]
This is going to conflict with #300

File: core/src/config_manager.cpp [hunk 1]
```diff
@@ -301,6 +301,11 @@ QString ConfigManager::serverName()
     return m_settings->value("Options/server_name", "An Unnamed Server").toString();
 }
 
+QString ConfigManager::serverNick()
+{
+    return m_settings->value("Options/server_nick", "").toString();
```
Comment (Salanto) [inline core/src/config_manager.cpp:306]:
```suggestion
    return m_settings->value("Options/server_nick", "NickName").toString();
``` 
If the key is missing we don't want an empty value, as that would look very silly when viewed in client.

**Review by Salanto — COMMENTED** [review]
Needs to be rebased to current master to be mergable, looks fine otherwise.
Make sure to add a test case to verify load behaviour.

**Comment by stonedDiscord** [comment]
isn't this the same as serverName or am i missing something here?

**Comment by mposs00** [comment]
closed as stale

**Pull request closed without merging**